osu.Game.Modes.Taiko/Objects/Drawables/DrawableHitCircle.cs

```diff
@@ -0,0 +1,196 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using OpenTK;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Modes.Taiko.Objects.Drawables.Pieces;
+using System;
+using System.Collections.Generic;
+using osu.Game.Modes.Objects.Drawables;
+using osu.Framework.Graphics;
+using osu.Framework.Input;
+using OpenTK.Input;
+using OpenTK.Graphics;
+using osu.Game.Graphics;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics.Transforms;
+
+namespace osu.Game.Modes.Taiko.Objects.Drawables
+{
+    public class DrawableHitCircleDon : DrawableHitCircle

```

Comment:
So what about LegacyReplay? DrawableOsuHitObject? Mod? Heck even Ruleset?
Or do those fall under the "will be fixed soon(tm)" category even though they're already in master?

Is this enforceable? Because maybe it should be, I do agree on enforcing one class/enum/struct per file in absolutely all cases (unless nested).